components/user_settings/display/user_settings_theme/custom_theme_chooser.jsx

```diff
@@ -13,6 +13,8 @@ import 'bootstrap-colorpicker';
 import Constants from 'utils/constants';
 import * as UserAgent from 'utils/user_agent';
 
+import Popover from 'components/widgets/popover/popover';
```

Comment:
Looks like a pattern on widget components but repeating '.../popover/popover' seems weird to me.  Could this be imported as `components/widgets/popover` by having index.ts (where it import then export that Popover  component)?